### PR TITLE
Feature/328 hdf format

### LINKED ISSUE #328: Add writer for HDF file format

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #328

### CODE REVIEW

File: pymchelper/writers/hdf.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import logging
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+class HdfWriter:
+    """
+    Supports writing HDF file format
```
Comment (nbassler) [inline pymchelper/writers/hdf.py]: Maybe you could add a few more info what this format is, and what uses it?

File: docs/hdf_converter.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.. highlight:: bash
+
+.. role:: bash(code)
+   :language: bash
+
+HDF file
+========
+
+Saves data in HDF binary file format. For every input detector a single file with :bash:`.h5` extension is written.
```
Comment (nbassler) [inline docs/hdf_converter.rst]: Maybe just mention an example program which uses this format.

Comment (grzanka) [reply docs/hdf_converter.rst]: It is done below, sample script reading HDF format is provided.

File: pymchelper/writers/hdf.py (hunk 1 above)
Comment (grzanka) [reply pymchelper/writers/hdf.py]: Done

**Pull request merged**